src/app/core/entity-list/duplicate-records/duplicate-records.service.spec.ts

```diff
@@ -41,6 +44,7 @@ describe("DuplicateRecordsService", () => {
         { provide: MatDialog, useValue: {} },
         { provide: MatSnackBar, useValue: {} },
         { provide: FileService, useValue: {} },
+        { provide: KeycloakAuthService, useValue: mockAuthService },
```

Comment:
does this really need the service?

Reply:
Yes, some other dependency needs Keycloak.